projects/uitpas/reference/uitpas.json

```diff
@@ -5794,21 +5777,558 @@
+                        "dateOfBirth": "2000-01-01",
+                        "registrationOrganizer": {
+                          "id": "abc12345"
+                        },
+                        "address": {
+                          "street": "Grote markt 12",
+                          "postalCode": "9300",
+                          "city": "Aalst",
+                          "country": "be"
+                        },
+                        "optInPreferences": {
+                          "serviceMails": true,
+                          "infoMails": true,
+                          "milestoneMails": true,
+                          "sms": false,
+                          "post": true
+                        },
+                        "nationality": "Belg",
+                        "legalTermsPaper": false,
+                        "legalTermsDigital": true,
+                        "parentalConsent": false,
+                        "registrationCardSystemId": 1
+                      }                      
+                    }
+                  }
+                }
+              }
+            }
+          },
+          "400": {
+            "description": "Bad Request. Possible error types:\n\n* https://api.publiq.be/probs/body/missing\n* https://api.publiq.be/probs/body/invalid-syntax\n* https://api.publiq.be/probs/body/invalid-data\n* https://api.publiq.be/probs/uitpas/organizer-not-found\n* https://api.publiq.be/probs/uitpas/cardsystem-not-found\n* https://api.publiq.be/probs/uitpas/invalid-city\n* https://api.publiq.be/probs/uitpas/invalid-insz-number\n* https://api.publiq.be/probs/uitpas/email-already-used\n\nThe detail property might include more information for the client developer.",
+            "content": {
+              "application/problem+json": {
+                "schema": {
+                  "$ref": "#/components/schemas/Error"
+                }
+              }
+            }
+          },
+          "401": {
+            "$ref": "#/components/responses/Unauthorized"
+          },
+          "403": {
+            "$ref": "#/components/responses/Forbidden"
+          }
+        },
+        "security": [
+          {
+            "USER_ACCESS_TOKEN": []
+          },
+          {
+            "CLIENT_ACCESS_TOKEN": []
+          },
+          {
+            "CLIENT_IDENTIFICATION": []
+          }
+        ],
+        "description": "Create an online order for one or more passholders.\n\nThis caller of this method, identified by client identification, client access token or user access token, must have ORDERS_CREATE permission.",
+        "requestBody": {
+          "content": {
+            "application/json": {
+              "schema": {
+                "$ref": "#/components/schemas/Order"
+              },
+              "examples": {
+                "Example": {
+                  "value": {
+                    "email": "example@example.org",
```

Comment:
@erwin1 is this correct.
That you also can pass an email address at the top level? Since I didn't see this in the fields of the documentation

Reply:
fixed